omero/users/cli/sessions.rst

```diff
@@ -76,7 +76,7 @@ Some of the options available to the :program:`omero login` command are:
 
     Set the password to use for the connection.
 
-.. option:: --sudo ADMINUSER, --sudo GROUPOWNER
+.. option:: --sudo ADMINUSER, GROUPOWNER
```

Comment:
The markup is wrong here.  GROUPOWNER is not part of the option.  You'll need two separate options, or `ADMINUSER|GROUPOWNER`